canonical-websites/snapcraft.io
# Create login blueprint

# Summary

Create login blueprint

# QA 

- `./run`
- http://127.0.0.1:8004/logout
- http://127.0.0.1:8004/account
- Login steps should be as usual
